pkg/controller/profilebundle/profilebundle_controller.go

```diff
@@ -120,12 +121,27 @@ func (r *ReconcileProfileBundle) Reconcile(request reconcile.Request) (reconcile
 	annotations := map[string]string{}
 	isISTag, isTagImageRef, err := r.pointsToISTag(instance.Spec.ContentImage)
 	if err != nil {
-		return reconcile.Result{}, err
+		if common.IsRetriable(err) {
+			return reconcile.Result{}, err
+		}
+
+		pbCopy := instance.DeepCopy()
+		pbCopy.Status.DataStreamStatus = compliancev1alpha1.DataStreamInvalid
+		pbCopy.Status.ErrorMessage = err.Error()
+		err = r.client.Status().Update(context.TODO(), pbCopy)
+		if err != nil {
+			reqLogger.Error(err, "Couldn't update ProfileBundle status")
+			return reconcile.Result{}, err
+		}
+		// this was a fatal error, don't requeue
+		return reconcile.Result{}, nil
```

Comment:
If there was a conflict with the Update() why do we not want to try again?

Reply:
oh, this comment is not applicable. Probably copy-pasted it from elsewhere. If there's a conflict it'll retry, which is done in the if statement above.

Reply:
ah yes, misread